sdk/spring/README.md

```diff
@@ -15,34 +15,36 @@ This repo is for Spring Boot Starters of Azure services. It helps Spring Boot de
-[azure-spring-boot-starter-keyvault-secrets][keyvault-starter-readme] | N/A | N/A | N/A | N/A
-[azure-spring-boot-starter-keyvault-certificates][keyvault-starter-certificates-readme] | N/A | N/A | N/A | N/A
 azure-active-directory-b2c-spring-boot-starter | [![Maven Central][aad-b2c-starter-mvn-2.3.x-img]][aad-b2c-starter-mvn-2.3.x] | [![Maven Central][aad-b2c-starter-mvn-2.2.x-img]][aad-b2c-starter-mvn-2.2.x] | [![Maven Central][aad-b2c-starter-mvn-2.1.x-img]][aad-b2c-starter-mvn-2.1.x] | N/A
-[azure-spring-boot-starter-active-directory-b2c][aad-b2c-starter-readme] | N/A | N/A | N/A | N/A
 azure-cosmosdb-spring-boot-starter | [![Maven Central][cosmosdb-starter-mvn-2.3.x-img]][cosmosdb-starter-mvn-2.3.x] | [![Maven Central][cosmosdb-starter-mvn-2.2.x-img]][cosmosdb-starter-mvn-2.2.x] | [![][cosmosdb-starter-mvn-2.1.x-img]][cosmosdb-starter-mvn-2.1.x] | [![][cosmosdb-starter-mvn-2.0.x-img]][cosmosdb-starter-mvn-2.0.x]
-[azure-spring-boot-starter-cosmos][cosmos-starter-readme] | N/A | N/A | N/A | N/A
 azure-mediaservices-spring-boot-starter | N/A | [![Maven Central][mediaservices-starter-mvn-2.2.x-img]][mediaservices-starter-mvn-2.2.x] | [![][mediaservices-starter-mvn-2.1.x-img]][mediaservices-starter-mvn-2.1.x] | [![][mediaservices-starter-mvn-2.0.x-img]][mediaservices-starter-mvn-2.0.x]
-[azure-spring-boot-starter-mediaservices][mediaservices-starter-readme] | N/A | N/A | N/A | N/A
 azure-servicebus-spring-boot-starter | N/A | [![Maven Central][servicebus-starter-mvn-2.2.x-img]][servicebus-starter-mvn-2.2.x] | [![][servicebus-starter-mvn-2.1.x-img]][servicebus-starter-mvn-2.1.x] | [![][servicebus-starter-mvn-2.0.x-img]][servicebus-starter-mvn-2.0.x]
-[azure-spring-boot-starter-servicebus][servicebus-starter-readme] | N/A | N/A | N/A | N/A
 spring-data-gremlin-boot-starter | [![Maven Central][gremlin-starter-mvn-2.3.x-img]][gremlin-starter-mvn-2.3.x] | [![Maven Central][gremlin-starter-mvn-2.2.x-img]][gremlin-starter-mvn-2.2.x] | [![][gremlin-starter-mvn-2.1.x-img]][gremlin-starter-mvn-2.1.x] | [![][gremlin-starter-mvn-2.0.x-img]][gremlin-starter-mvn-2.0.x]
-[azure-spring-boot-starter-data-gremlin][gremlin-starter-readme] | N/A | N/A | N/A | N/A
 azure-spring-boot-metrics-starter | [![Maven Central][metrics-starter-mvn-2.3.x-img]][metrics-starter-mvn-2.3.x] | [![Maven Central][metrics-starter-mvn-2.2.x-img]][metrics-starter-mvn-2.2.x] | [![][metrics-starter-mvn-2.1.x-img]][metrics-starter-mvn-2.1.x] | [![][metrics-starter-mvn-2.0.x-img]][metrics-starter-mvn-2.0.x]
-[azure-spring-boot-starter-metrics][metrics-starter-readme] | N/A | N/A | N/A | N/A
 azure-servicebus-jms-spring-boot-starter | [![Maven Central][jms-starter-mvn-2.3.x-img]][jms-starter-mvn-2.3.x] | [![Maven Central][jms-starter-mvn-2.2.x-img]][jms-starter-mvn-2.2.x] | [![Maven Central][jms-starter-mvn-2.1.x-img]][jms-starter-mvn-2.1.x] | N/A
```

Comment:
Sort lines by artifactId.

Reply:
How about we not sorting lines of Spring Cloud tables given that there are starter, stream binder and integration packages of each service, And the current order lists packages towards services: evenuhubs, servicebus, storage, cache and appconfig, which we try to list packages with the popularity order.

Reply:
OK